elasticsearch.md

```diff
@@ -2118,28 +2123,31 @@ Usually, we would like to configure three main things for defining the way LDAP
     * `bind_dn` (string, optional, default: [not present]) - a username used to connect to the LDAP service. We can skip this setting when our LDAP service allows for anonymous binding
     * `bind_password` (string, optional, default: [not present]) - a password used to connect to the LDAP service. We can skip this setting when our LDAP service allows for anonymous binding
 
-2. a way to **search users**. In ROR it can be done using the following YAML keys (used by all LDAP rules):
+2. a way to **search users**. In ROR it can be done using the following YAML keys (under the `users` section) (used by all LDAP rules):
    * `search_user_base_DN` (string, required) - should refer to the base Distinguished Name of the users to be authenticated
    * `user_id_attribute` (string, optional, default: `uid`) - should refer to a unique ID for the user within the base DN
    * `skip_user_search` (boolean, optional, default: `false`) - when you set `user_id_attribute: "cn"` you may want to skip the user search. This optimizes the authentication, which is done in two steps (searching for a user DN and authenticating the user with a given DN). If you configure it to be `true`, the user's DN will be `cn={user_login},{search_user_base_DN}`.
 
-3. a way to **search user groups** (NOT used by [`ldap_authentication`](#ldap_authentication) rule). In ROR, depending on LDAP schema, a relation between users and groups can be defined in:
-   1. Group entry - it has an attribute that refers to User entries (this is the default):
-      * `search_groups_base_DN` (required) - should refer to the base Distinguished Name of the groups to which these users may belong
-      * `group_name_attribute` (string, optional, default: `cn`) - is the LDAP group object attribute that contains the names of the ROR groups
-      * `unique_member_attribute` (string, optional, default: `uniqueMember`) - is the LDAP group object attribute that contains the names of the ROR groups
+3. a way to **search user groups** (NOT used by [`ldap_authentication`](#ldap_authentication) rule). You can configure all properties under the `groups` section in LDAP connector configuration.
+   
+   In ROR, depending on LDAP schema, a relation between users and groups can be defined in:
+   1. Group entry - it has an attribute that refers to User entries (`mode: search_groups_in_group_entries`this is the default):
+      * `search_groups_base_DN` (required) - should refer to the base Distinguished ID of the groups to which these users may belong
+      * `group_id_attribute` (string, optional, default: `cn`) - is the LDAP group object attribute that contains the IDs of the ROR groups
+      * `group_name_attribute` (string, optional, default: group_id_attribute) - is the LDAP group object attribute that contains the [name](details/structured-groups.md) of the ROR groups
```

Comment:
IMO the link should be added also in case of other authz rules

Reply:
What rules did you have in mind? I added the link in the External Authorization, LDAP, Jwt, and local groups sections